lively.ide/js/inspector/index.js

```diff
@@ -464,7 +465,7 @@ export class PropertyControl extends DraggableTreeLabel {
     const inspector = tree.owner;
     const handler = async (evt) => {
       // if already open, return
-      const editor = part(ShadowPopup, { hasFixedPosition: true });
+      const editor = part(ShadowPopupLight, { epiMorph: true, hasFixedPosition: true });
```

Comment:
I might be missing somethin obvious here, but shouldn't we just set this to be true for all popups at all times? :thinking: